### DISCUSSION TITLE
Warnings in the log

### DISCUSSION THREAD

**Discussion by clee604** [KrystianD/bm2-battery-monitor#7] [Q&A]
I'm not sure if this warning came up recently in an OTA update or its been always there and I just didn't see it in the logs.

I've been getting this error in the logs:
`[ble_sensor:080] Error reading char at handle 46, status=2`

The voltage sensor is still outputting the correct values so it still works, but am I missing some configuration in my ESPHome or Home Assistant?

Thanks in advance

**Comment by KrystianD** [comment]
Hi, unfortunately I am not able to help with this. See https://github.com/KrystianD/bm2-battery-monitor/issues/3

**Comment by clee604** [comment]
Thanks, not sure if it even a  problem since voltage still works. Though now it seems the bm2.yaml include file got removed in my HA, not sure if the recent HA BLE update did something.... i can't seem to find my bm2.yaml config anywhere
